scripts/common/common.sh

```diff
@@ -709,3 +706,12 @@ MACHINE_ID=
 function get_machine_id() {
     MACHINE_ID="$(${DIR}/bin/kurl host protectedid || true)"
 }
+
+function canPrompt() {
```

Comment:
can you make this snake case?

Reply:
Np. There are some other places we'll need to make these changes too like `prompts.sh`

Reply:
Ya, I think for all new code we should stick to snake